api/endpoints/off_topic_channel_names/endpoints.py

```diff
@@ -0,0 +1,142 @@
+from typing import Optional, Union
+
+from fastapi import APIRouter, Depends
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Query, Session
+
+from api.core.database.models.api.bot import OffTopicChannelName
+from api.core.schemas import ErrorMessage
+from api.endpoints.dependencies.database import create_database_session
+
+otn = APIRouter(prefix="/off-topic-channel-names")
+
+
+def get_all_otn(db_session: Session) -> Query:
+    """Get a partial query object with .all()."""
+    return db_session.query(OffTopicChannelName).all()
```

Comment:
Could you move this to a separate file, into something like`otn_dependency`?

Reply:
It isn't really a dependency and is only used in the current file, I don't think it deserves to go in a separate dependency file.

Reply:
If your code depends on this function, then it is a dependency, the reason I ask is because in the future new dependencies could be added, and I believe it would make the project structure more friendly.